FWCore/Modules/src/EventContentAnalyzer.cc

```diff
@@ -472,6 +509,8 @@ namespace edm {
      np = desc.addOptionalUntracked<bool>("listContent", true);
      np->setComment("If true then print a list of all the event content.");
 
+     np = desc.addOptionalUntracked<bool>("listProvenance", false);
```

Comment:
@Dr15Jones Is there a good reason for these to be optional, given that all of them have default values? Now the defaults are specified twice.